addons/mail/static/src/new/core/messaging_service.js

```diff
@@ -119,7 +119,7 @@ export class Messaging {
             if (data.current_partner) {
                 this.partner.insert(data.current_partner);
             }
-            if (data.currentGuest) {
+            if (data.currentGuest?.id) {
```

Comment:
`data.currentGuest` is false when not guest. I don't see reason to inspect `id`

Reply:
but we will get data = ['clear'], which will lead into creating a guest

Reply:
I don't receive `['clear']`, and I don't see any such occurrences in the code